integration/spark2/src/main/scala/org/apache/spark/sql/execution/datasources/SparkCarbonTableFormat.scala

```diff
@@ -175,6 +172,11 @@ with Serializable {
           dataSchema: StructType,
           context: TaskAttemptContext): OutputWriter = {
         val model = CarbonTableOutputFormat.getLoadModel(context.getConfiguration)
+        val appName = context.getConfiguration.get(CarbonCommonConstants.CARBON_WRITTEN_BY_APPNAME)
+        if (null != appName) {
```

Comment:
Is there is no appName, I think we should construct one, the appName should be always write into the file

Reply:
actually the appname will be always set ,spark will always set the appname, this check is added for one of the test cases , i will remove this